module/main.c

```diff
@@ -131,13 +127,13 @@ int16_t output, output_new;
 #define FIRSTRUN_KEY 0x22
 
 typedef const struct {
-    tele_script_t script[10];
-    tele_pattern_t patterns[4];
+    scene_script_t script[10];
```

Comment:
yeah as namespacing i think it's a good thing. the only concern would be if for whatever reason scripts and patters as entities could live outside of scenes, but perhaps that would be a different type altogether.